frontend/js/restqueries.js

```diff
@@ -1,4 +1,4 @@
-function retreiveFromDB(collection, query, populateCallback) {
+function retreiveFromDB(collection, query, populateCallback) {//additional send true false boolean parameter to fuse DB queries?
```

Comment:
I was thinking of fusing sendCommandToDB with out retrieve command and add a true false boolean to check whether it was sending or receiving. Thinking about it more it would probably just clutter up a single function.